### ISSUE TITLE
IOS: Module 'AirshipKit' not found

### ISSUE THREAD

**Issue by erez-guesty** [urbanairship/react-native-module#101]
Xcode 9.2

I follow Carthage setup method: https://github.com/urbanairship/ios-library
> $ brew update
$ brew install carthage

Cartfile
> github "urbanairship/ios-library"

then ran command 
> $ carthage update

then follow Carhage instructions:

If you're building for iOS, tvOS, or watchOS

3. On your application targets’ General settings tab, in the “Linked Frameworks and Libraries” section, drag and drop each framework you want to use from the Carthage/Build folder on disk.

4. On your application targets’ Build Phases settings tab, click the + icon and choose New Run Script Phase. Create a Run Script in which you specify your shell (ex: /bin/sh), add the following contents to the script area below the shell:

> /usr/local/bin/carthage copy-frameworks

5. Add the paths to the frameworks you want to use under “Input Files”, e.g.:
>$(SRCROOT)/Carthage/Build/iOS/Result.framework
$(SRCROOT)/Carthage/Build/iOS/ReactiveSwift.framework
$(SRCROOT)/Carthage/Build/iOS/ReactiveCocoa.framework

6. Add the paths to the copied frameworks to the “Output Files”, e.g.:
> $(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/Result.framework
$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/ReactiveSwift.framework
$(BUILT_PRODUCTS_DIR)/$(FRAMEWORKS_FOLDER_PATH)/ReactiveCocoa.framework

![image](https://user-images.githubusercontent.com/33092944/38235647-83cad752-372a-11e8-9ce1-2c74397fe29b.png)

![image](https://user-images.githubusercontent.com/33092944/38236229-2da9bb98-372c-11e8-9275-6e598d191826.png)

![image](https://user-images.githubusercontent.com/33092944/38236249-3b186798-372c-11e8-8b91-7cea6ae13e56.png)

any ideas ??

**Comment by rlepinski** [comment]
@erez-guesty what version of carthage are you using? My sample works but I am on .24. Coworkes sample does not and he is running the latest Carthage. Not sure what changed in Carthage or if its just my particular setup that is making it work, but we are looking into it.

**Comment by MaxInMoon** [comment]
Same here:
"framework not found AirshipKit" when building (I used cocoa pods)

**Comment by BrianBatchelder** [comment]
@erez-guesty: in Xcode, please try adding the following build setting to your app's project using Editor -> Add Build Setting  -> Add User-Defined Setting:
```
Key = AIRSHIP_SEARCH_PATH
Value = $(SRCROOT)/../../../../AirshipSample/ios/Carthage/Build/iOS
```

**Comment by BrianBatchelder** [comment]
@erez-guesty: in Xcode, please try adding the following build setting to the UARTCModule sub-project's Build Settings using Editor -> Add Build Setting  -> Add User-Defined Setting:
```
Key = AIRSHIP_SEARCH_PATH
Value = $(SRCROOT)/../../../../AirshipSample/ios/Carthage/Build/iOS
```

**Comment by BrianBatchelder** [comment]
@MaxInMoon : since this issue is a Carthage issue, not a CocoaPods issue, could you please check your steps and file a new issue if the problem persists?

**Comment by MaxInMoon** [comment]
@rlepinski @BrianBatchelder thanks for your time.
To bring feedback to my prvious message, my "CocoaPods" issue was that I was open Xcode project with `myapp.xcodeproj` instead of  `myapp.xcworkspace`.

**Comment by BrianBatchelder** [comment]
@erez-guesty : This issue has been fixed in [Release 1.6.1](https://www.npmjs.com/package/urbanairship-react-native).
